src/libapi.mjs.d.ts

```diff
@@ -55,6 +58,27 @@ declare class Context {
   isDeleted(): boolean;
 }
 
+export declare class Camera {
```

Comment:
Camera is not part of libapi.mjs though. This `.d.ts` is only to describe types in the Emscripten-generated `libapi.mjs`.

For `camera.js` you can describe types inline in JSDoc comments instead so that they're colocated with code.

Reply:
Question because I'm not used to JSDoc: Can you still get type hints in typescript? I'm testing right now after adding the JSDoc syntax to the Camera class in `camera.js` and removing the definition from `libapi.d.ts` but type hints no longer work.

I used to do:
```ts
import Camera from 'web-gphoto2-npm-test';
import type { Camera as CameraClass } from 'web-gphoto2-npm-test';

export const camera: CameraClass = new Camera();
```
![image](https://github.com/GoogleChromeLabs/web-gphoto2/assets/37150873/1994fa76-86df-4b7c-b2ef-3437feefc575)


But right now it tells me that the package does not export a type called Camera, and removing the type definition all together makes using the class a bit more difficult because the functions are not showing up as type hints:

![image](https://github.com/GoogleChromeLabs/web-gphoto2/assets/37150873/d23b9169-8f29-41e6-ac89-7db48360f89b)

Reply:
Yes, they're supposed to work, both hints and actual typechecking. If you can't get it to work I might take a look later.